packages/@atjson/document/src/index.ts

```diff
@@ -375,6 +375,63 @@ export default class Document {
     return matches;
   }
 
+  canonical(): Document {
```

Comment:
We can remove the type from here— we actually _don't_ want this to return a `Document` type, but the type of `this`, which I think `clone` already does.

(this will happen already using type inference)